.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build and Push Docker Image
+        uses: Dwolla/jenkins-agents-workflow/.github/actions/build@main
+        with:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+          BASE_TAG: <my-specified-base-image-tag>
```

Comment:
@gorskilauren is this `BASE_TAG` necessary when it's not a Jenkins image?

Reply:
Technically, yes it is defaulted to `true` in the upstream action [here](https://github.com/Dwolla/jenkins-agents-workflow/blob/08df2cd11f4154d8ad83b94f711b718bbd47c5ef/.github/actions/build/action.yml#L15), since that's how we've used it so far!

FWIW I think we use that `BASE_TAG` param to derive our own Dockerhub image tag [here](https://github.com/Dwolla/jenkins-agents-workflow/blob/08df2cd11f4154d8ad83b94f711b718bbd47c5ef/.github/actions/build/action.yml#L64). If we're ok taking that out for non-agents, we could probably take out the BASE_TAG (and TAG_NAME) concepts from the action!

Reply:
(And by taking out the concepts, I really mean keeping them but as "optional" instead of "required"!)

Reply:
That makes sense. IMO using it to build our Docker Hub tag still makes sense, but it would be nice if the value could come from the Dockerfile so that dependabot (or whatever) can keep the upstream tag up to date. This works for now though.